flax/linen/module.py

```diff
@@ -809,7 +809,8 @@ def apply(self, variables: VariableDict, *args, rngs: RNGSequences = None,
       variables: A dictionary containing variables keyed by variable
         collections. See :mod:`flax.core.variables` for more details
         about variables.
-      rngs: The rngs for the variable collections.
+      rngs: The rngs for the variable collections. For example :class:`flax.linen.stochastic.Dropout`
```

Comment:
Small suggestion: I'd recommend using "RNG" instead of "rng" for consistency, since it's an acronym